.circleci/config.yml

```diff
@@ -469,7 +469,7 @@ jobs:
             latest_oss_commit="$(git rev-parse origin/master)"
             # adding two merge messages will add them into different paragraphs which will achieve what is in
```

Comment:
should these comments above be removed too?

Reply:
good catch, removed!